packages/components/src/progress-bar/README.md

```diff
@@ -1,13 +1,42 @@
 # ProgressBar
 
-<div class="callout callout-alert">
-This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.
-</div>
-
 A simple horizontal progress bar component.
 
 Supports two modes: determinate and indeterminate. A progress bar is determinate when a specific progress value has been specified (from 0 to 100), and indeterminate when a value hasn't been specified.
 
+## Usage
+
+Basic usage:
+
+```jsx
+import { ProgressBar } from '@wordpress/components';
+
+const MyLoadingComponent = () => {
+  return <ProgressBar />
```

Comment:
We'll need a semicolon and use tabs for this to be valid syntax according to ESLint and Prettier:

```suggestion
	return <ProgressBar />;
```

Reply:
Sounds good. As a related questio, do you have ESLint/prettier activated for JSX code blocks in markdown?

Reply:
Yup, I do.